uiowa/uiowa
# facilities post launch

tracks https://github.com/uiowa/uiowa/issues/7607

## To test

- Code review

```
ddev blt frontend && ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local uli /art-campus/art-campus-gallery
```

1. Sync site
2. [Go here and](https://facilities.uiowa.ddev.site/art-campus/art-campus-gallery) see that the `Status` filter is autopopulated with `Active`, and that the view is autofiltered.
3. Also observe on that same page that the `Status` filter is at the bottom of the filter list.
4. Go find an artist and check that the photo is gone, and that the website is moved under the birth/death year fields.
    - Also checking the content type display page would be good to make sure that the sidebar is completely removed, and all fields no longer sit in a different section away from the website.
5. Go to a piece of art, and observe that the following fields are in the following order:
    - Artist (1st)
    - Artist website (2nd)
    - Building (3rd)
6. [Go here and](https://facilities.uiowa.ddev.site/buildings) see that the filter now autosubmits.
7. Masquerade as a collaborator and check that you can now delete and edit any alert.

## Icon updates

`fa-ul` and `fa-li` classes have been replaced by the `fa-field-item` wrapper class and targeted CSS for positioning the font awesome icon absolutely outside of the field text.  Percy shows that events, article, people teasers are the same but click through any other icon heavy sites like classrooms or alert teasers if you want to test this out further.  


## Person field label updates

Person field labels have been updated to be visually hidden.  You should see a number of updates to `core.entity_view_display.node.person.default.yml` here.
